test/test_time_stepping.jl

```diff
@@ -46,9 +53,22 @@ function incompressible_in_time(arch, ft, Nt)
     # Just add a temperature perturbation so we get some velocity field.
     @. model.tracers.T.data[8:24, 8:24, 8:24] += 0.01

```

Comment:
I think it would be better to add a velocity fluctuation.

Reply:
Ah maybe but then you'd have to ensure that the initial velocity field is divergence-free. Doable but makes the test a bit more complex. A cube-shaped temperature anomaly might not have been the best choice.

Reply:
Ok. But you should set g and alpha then to make sure the test doesn't change if the defaults are changed somewhere.